s3/README.md

```diff
@@ -1,16 +1,117 @@
-# S3
+# AWS S3 (Simple Storage Service)
 
-## Args
+Create a S3 bucket as an Acorn with a single click or command.
+
+## Usage
+
+From the CLI you can run the following command to create a S3 bucket.
 
+```shell
+acorn run -n s3-bucket ghcr.io/acorn-io/s3:v0.#.#
 ```
 
-Volumes:   <none>
-Secrets:   <none>
-Containers: <none>
-Ports:     <none>
+From an Acornfile you can create the bucket using the acorn too.
+```cue
+services: s3: {
+	image: "ghcr.io/acorn-io/s3:v0.#.#"
+}
+containers: app: {
+	build: context: "./"
+	ports: publish: ["8080/http"]
+	env: {
+        consumes: ["s3"]
+        BUCKET_URL: "@{service.s3.data.url}"
```

Comment:
whitespace formatting is off for env
consumes should be out of the env block

Reply:
Looks fixed now. It seemed correct locally but I recreated the indentation to try and fix it here too.